transform/snowflake-dbt/models/hightouch/freemium/cloud_starter/cs_sso_facts.sql

```diff
@@ -53,17 +54,19 @@ with existing_leads as (
     false as marketing_suspend,
     lead.sfid as lead_sfid,
     'SSO' as signup_method,
+    coalesce(lead.LEAD_SOURCE_TEXT__C,'Referral') as lead_source,
+    coalesce(lead.LEAD_SOURCE_DETAIL__C,'Mattermost Cloud') as lead_source_detail,
     case
-        when subscriptions.cws_dns is not null then 'Completed Signup'
+        when subscriptions.cws_dns is not null then 'Workspace Created'
         else 'Account Created' 
     end as campaign_status
     from 
         {{ ref('customers') }}
         left join {{ ref('subscriptions') }} on customers.id = subscriptions.customer
+        left join {{ ref('products') }} on products.id = coalesce(subscriptions.plan:"product"::varchar, subscriptions.metadata:"current_product_id"::varchar)
         left join existing_leads as lead on customers.email = lead.email and lead.row_num = 1
         left join existing_contacts as contact on customers.email = contact.email and contact.row_num = 1
         left join existing_members as campaignmember on customers.email = campaignmember.email and campaignmember.row_num = 1
-        where customers.email not in ( select email from {{ ref('cs_signup_campaign')}} )
-        and lower(subscriptions.edition) = 'cloud starter' and lead_sync_eligible 
+        where customers.email not in ( select email from {{ ref('cs_signup_campaign_dev')}} ) and lead_sync_eligible 
 )
-select * from sso_facts_pre
\ No newline at end of file
+select * from sso_facts_pre where lower(edition) = 'cloud starter' 
```

Comment:
@munish7771 - is it `subscriptions.edition` (like before) or only `edition`

Reply:
column is from same cte sso_facts_pre, reference is not needed. only edition.